src/Users/Views/user_settings.php

```diff
@@ -120,8 +120,8 @@
                     <div class="col-12 col-sm-4">
                         <div class="form-check">
                             <input class="form-check-input" type="checkbox" name="email2FA"
-                                   value="Shield\Authentication\Actions\Email2FA" id="email-2fa"
-                                <?php if (old('email2FA', setting('Auth.actions')['login']) === 'Shield\Authentication\Actions\Email2FA') : ?>
+                                   value="Bonfire\Auth\Authentication\Actions\Email2FA" id="email-2fa"
```

Comment:
As per comments above, I'm not sure this is really needed and would require a maintenance headache.

Reply:
this is related to Themeable trait end setting templates